pkg/cloud/api/type.go

```diff
@@ -20,6 +20,15 @@ import (
 	"reflect"
 )
 
+// UntypedResourceType should be used to represent missing GCE resources
+// in type traits functions.
+// Every GCE resource has Name and SelfLink so check schema for this
+// object will pass.
```

Comment:
give a usage example. 

Use 'fill paragraph' to format the comments (fill to 78 columns). Note: this is M-q in Emacs, 'gqap' in Vim.